src/styles/variables.sass

```diff
@@ -0,0 +1,49 @@
+// Favorite constants
+$gratio: 1.618
+$pi: 3.14159
+
+// Dimensions
+$radius: 2px
+$spacing: 24px
+$padding: 10px
+
+// Font properties
+$font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif
+$font-family-serif: Georgia, Cambria, 'Times New Roman', Times, serif
+$font-family-monospace: Monaco, Menlo, Consolas, 'Courier New', monospace
+
+$font-family-base: $font-family-sans-serif
+$font-family-head: $font-family-serif
+
+$base-font-size: 16px
+$fw-normal: 400
+$fw-medium: 500
+$fw-semibold: 600
+$fw-bold: 700
+$em-base: $base-font-size
+
+// Colors
+$c-main: green
```

Comment:
Mě proměnné na barvy taky nevadí, než začnu reálně kodovat tak si přesně do `variables.styl` definuju fonty, barvy, radius atd...různý barvy řeším jako `c-primary` nebo `c-secondary` a jejich varianty prefixem `-light` nebo `-dark` a na to zpravidla pouzivam mixin např. `lighten($c-primary, 10%)`. Víc základních barev než 2 se většinou nepoužívají, a pokud pro nějaký komponenty jo tak pro dals ibarvy pouzivam `c-red` apod. Jinak prefix `-alt` taky moc nepouzivam, pokud jo tak je to na kontrastni barvu.